sdcm/sct_config.py

```diff
@@ -1426,6 +1426,9 @@ class SCTConfiguration(dict):
              help="""add/remove num of process on each round"""),
         dict(name="use_hdr_cs_histogram", env="SCT_USE_HDR_CS_HISTOGRAM", type=boolean,
              help="""Enable hdr histogram logging for cs"""),
+
+        dict(name="stop_on_hw_perf_failure", env="SCT_STOP_ON_HW_PERF_FAILURE", type=boolean,
+             help="""Stop performance test if hardware performance test failed""")
```

Comment:
The description isn't good enough. please explain what is considered as failed, if it's within margin and what's the margin and refer to the relevant params.

Reply:
Update description